lib/llvmopencl/LLVMUtils.h

```diff
@@ -49,6 +49,7 @@ void
 regenerate_kernel_metadata(llvm::Module &M, FunctionMapping &kernels);
 
 // Remove a function from a module, along with all callsites.
+POCL_EXPORT
 void eraseFunctionAndCallers(llvm::Function *Function);
```

Comment:
I'm wondering should we add pocl_ prefix to this libpocl.so exported function for unity?

Reply:
This is a C++ symbol in pocl namespace, so I'm fine with it.
`_ZN4pocl23eraseFunctionAndCallersEPN4llvm8FunctionE@Base`

But I found some more symbols needed by the accel device, just added a commit to export them.
These don't fit the naming scheme and should probably be renamed: `alloc_buffer_from_region`, `free_chunk`, `init_mem_region`. Especially the second one sounds generic enough to allow clashes with other libraries.

That leaves only one already exported symbol not matching the naming scheme: `get_llvm_cpu_name`